src/storage/metrics.rs

```diff
@@ -63,6 +56,30 @@ lazy_static! {
             exponential_buckets(0.0005, 2.0, 20).unwrap()
         ).unwrap();
 
+    pub static ref SCHED_GET_SNAPSHOT_HISTOGRAM_VEC: HistogramVec =
```

Comment:
Can this hurt performance?

Reply:
maybe storage async snapshot metric is enough?